charts/capi-providers-config/templates/core-provider.yaml

```diff
@@ -21,7 +21,7 @@ data:
     {{- if .Values.core.additionalManifests }}
     {{- toYaml .Values.core.additionalManifests.manifests | nindent 4 }}
     {{- end }}
-    {{- if .Values.metrics.enabled }}
+    {{- if $.Values.metrics.enabled }}
```

Comment:
other references to .Values do not have to use `$`
can you explain why it was needed? Helm chart didn't recognize this option in values.yaml?